### PR TITLE
Add "Abandon Jenkins for Great Good!" post.

### PR DESCRIPTION
**Pull request by arturdryomov**
@artem-zinnatullin, PTAL.

### CODE REVIEW

**Comment by arturdryomov** [comment]
@artem-zinnatullin, if you don’t have time, can I go ahead without your insight?

File: content/posts/2018-12-09-abandon-jenkins.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+might arise from each one of them. Gradle compile times suffer?
+Need to profile, compare different Gradle versions, check build scripts,
+analyze Java and Kotlin compilers and find out that it was
+a [Linux kernel feature](https://lkml.org/lkml/2018/11/19/37) or a SSD failure.
+Sounds fun, right? Don’t get me started on how much time these things consume.
+Imagine a weird code issue which takes hour after hour and multiply
+it by a factor of uncontrolled external tools and environments.
+
+> :warning: I have to note this explicitly — I’m describing a situation related to small teams.
+> Big companies have dedicated units to manage all of this and not a single person.
```
Comment (artem-zinnatullin) [inline content/posts/2018-12-09-abandon-jenkins.md:38]: heh, I wish lol

File: content/posts/2018-12-09-abandon-jenkins.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+It was a natural choice for the team because it is the only service that works
+with [Bitbucket Server](https://www.atlassian.com/software/bitbucket/server)
+([not the same thing as the cloud version](https://confluence.atlassian.com/confeval/development-tools-evaluator-resources/bitbucket/bitbucket-cloud-vs-server)).
+Interesting fact — Bitbucket Server support
+[was added by an open source contributor](https://github.com/bitrise-io/bitrise-webhooks/pull/67).
+It is possible because Bitrise components are [open source](https://github.com/bitrise-io).
+
+[Pricing is reasonable](https://www.bitrise.io/pricing). I like that
+the hardware configuration is not hidden away from customers.
+At the same time there is an opt-in for Elite machines with better specs.
```
Comment (artem-zinnatullin) [inline content/posts/2018-12-09-abandon-jenkins.md]: probably worth putting Elite in quotes

File: content/posts/2018-12-09-abandon-jenkins.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+ release:
+    steps:
+    - git-clone:
+        title: "Clone Git repository."
+    - cache-pull:
+        title: "Pull cache."
+    - script:
+        title: "Run."
+        inputs:
+        - content: "bash ci/jobs/release.sh"
```
Comment (artem-zinnatullin) [inline content/posts/2018-12-09-abandon-jenkins.md:167]: nit: ideally, this line wouldn't be calling `bash` directly, script can define expected interpreter

Comment (artem-zinnatullin) [reply content/posts/2018-12-09-abandon-jenkins.md:167]: Also scripts don't need `.sh`, but that's subjective (no need to address, just commenting)

File: content/posts/2018-12-09-abandon-jenkins.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        title: "Attach artifacts."
+        is_always_run: true
+        inputs:
+        - deploy_path: "artifacts/"
+        - is_enable_public_page: false
+```
+
+The transition from Jenkins to Bitrise was seamless. For a while we used
+both in parallel and after a month I’ve quetly shut down Jenkins with no
+consequences for the team.
```
Comment (artem-zinnatullin) [inline content/posts/2018-12-09-abandon-jenkins.md:178]: What about automatic merging and PR rebuilds on target branch change?

**Review by artem-zinnatullin — APPROVED** [review]

**Pull request merged**